docs/app_management.md

```diff
@@ -10,9 +10,11 @@ We'll use this information to setup your team members on the app stores.
 
 ## Managing user access to App Store Connect
 
-[App Store Connect](https://appstoreconnect.apple.com/login) is the platform that allows teams to manage their apps on the Apple App Store. By assigning roles to users, you control who has access to do certain functions. Roles available to app team members are: 
+[App Store Connect](https://appstoreconnect.apple.com/login) is the platform that allows teams to manage their apps on the Apple App Store. By assigning roles to users, you control who has access to do certain functions. 
 
+A person may have more than one role. For example, a user may have both the App Manager and Customer Support roles. Please consult Apple's [Program Roles](https://developer.apple.com/support/roles/) documentation for the functions available to each role. 
 
+Roles available to app team members are: 
 
```

Comment:
@CoffeeGurlNia - I reorganized the section a bit. I've moved the section about multiple roles to above the list of roles. I think it should be separate from the Apple ID section